### ISSUE TITLE
ds-references stripped from production?

### ISSUE THREAD

**Issue by richmolj** [emberjs/data#4216]
I'm using the new `ds-references` feature flag from ember-data 2.4. Everything works in development, but in production it looks like the code is not being applied:

From console:

```
> EmberENV.FEATURES
>> Object {ds-references: true}
> model._internalModel.referenceFor
>> undefined
```

**Comment by bmac** [comment]
Feature flags that are not turned on by default (like `ds-references`) are only suppose to be accessible on the canary branch. I believe the fact that `ds-references` is available in development is a bug.

**Comment by richmolj** [comment]
There is a warning about this when I run 2.4 in development.

**Comment by kernel-io** [comment]
provided that this is still behind a feature flag, is there any safe way to get only the id of the relationship with 2.4?

**Comment by pangratz** [comment]
As @bmac pointed out, `ds-references` is not yet available in a release of `ember-data`. Currently it's available in `canary` and `2.5-beta` series. It will most likely be included when `2.5` is released.

---

> There is a warning about this when I run 2.4 in development.

@richmolj this has been fixed upstream in Ember.js, see #4213 for reference.

---

> provided that this is still behind a feature flag, is there any safe way to get only the id of the relationship with 2.4?
